### PR TITLE
fix: stake page ui fix

### PR DESCRIPTION
**Pull request by canhamzacode**

### CODE REVIEW

File: frontend/src/pages/vault/stake/Stake.jsx [hunk 1]
```diff
@@ -40,86 +40,89 @@ function Stake() {
...
+                  <div className="apy-rate">$0.00 APY / year</div>
+                  <GasFee />
+                </div>
 
-         </div>
-          <Button variant="secondary" size="lg" className="stake-button">
-            Stake
-          </Button>
+            </div>
+            <Button variant="secondary" size="lg" className="stake-button">
```
Comment (whateverfw) [inline frontend/src/pages/vault/stake/Stake.jsx:122]: button styles are broken now

File: frontend/src/pages/vault/stake/stake.css [hunk 2]
```diff
@@ -43,7 +43,29 @@
...
+  margin-top: 36px;
+}
+
+.stake-header-text{
+  font-size: 24px;
+  font-weight: 600;
+  color: var(--primary);
+  text-align: center;
+  /* margin-top: 30px; */
+  /* background: #000; */
```
Comment (whateverfw) [inline frontend/src/pages/vault/stake/stake.css:56]: don't leave commented code, remove it

File: frontend/src/pages/vault/stake/stake.css [hunk 3]
```diff
@@ -191,6 +213,11 @@
   width: 100%;
 }
 
+.button--secondary::before {
+  border-radius: 8px !important;
```
Comment (whateverfw) [inline frontend/src/pages/vault/stake/stake.css:217]: you should almost never use !important in css, remove it

File: frontend/src/pages/vault/stake/stake.css
Comment (whateverfw) [inline frontend/src/pages/vault/stake/stake.css:1]: sidebar is broken and do not stick to the top when scrolled

Comment (whateverfw) [inline frontend/src/pages/vault/stake/stake.css:1]: table and mobile pages  are completely broken, overflowed by a sidebar, did you even test it?

File: frontend/src/components/StakeCard/metricCard.css [hunk 4]
```diff
@@ -1,12 +1,20 @@
 .stake-card {
-  width: 309px;
+  /* width: 309px; */
   background: var(--header-button-bg);
   border: var(--midnight-purple-border);
-  border-radius: 900px;
+  border-radius: 8px !important;
```
Comment (whateverfw) [inline frontend/src/components/StakeCard/metricCard.css:5]: avoid !important

File: frontend/src/components/StakeCard/metricCard.css [hunk 5]
```diff
@@ -1,12 +1,20 @@
 .stake-card {
-  width: 309px;
+  /* width: 309px; */
```
Comment (whateverfw) [inline frontend/src/components/StakeCard/metricCard.css:2]: why did you remove this?

File: frontend/src/components/StakeCard/metricCard.css [hunk 6]
```diff
@@ -1,12 +1,20 @@
...
   flex-direction: column;
   justify-content: center;
   align-items: center;
+  height: 101px;
+}
+
+.stake-card .label{
+  font-size: 14px;
+  font-weight: 600;
+  color: #83919F;
```
Comment (whateverfw) [inline frontend/src/components/StakeCard/metricCard.css:16]: use css variables for colors

**Review by whateverfw — CHANGES_REQUESTED** [review]

File: frontend/src/components/StakeCard/metricCard.css (hunk 5 above)
Comment (canhamzacode) [reply frontend/src/components/StakeCard/metricCard.css:2]: There was no difference in having it there or removing it. That's why i removed it

File: frontend/src/pages/vault/stake/stake.css
Comment (canhamzacode) [reply frontend/src/pages/vault/stake/stake.css:1]: None of my change is meant to affect the sidebar. noticed that but i didnt touch it

File: frontend/src/pages/vault/stake/Stake.jsx (hunk 1 above)
Comment (canhamzacode) [reply frontend/src/pages/vault/stake/Stake.jsx:122]: Do you mean all buttons or only this. The button style is making it difficult to style this

File: frontend/src/pages/vault/stake/stake.css
Comment (canhamzacode) [reply frontend/src/pages/vault/stake/stake.css:1]: Yes i did. but that was not because of my implementation. Mine was just to update the ui to fit the figma. I believe a seperate issue can be created for that

Comment (whateverfw) [reply frontend/src/pages/vault/stake/stake.css:1]: Sidebar is a part of a page, please fix it.

Comment (whateverfw) [reply frontend/src/pages/vault/stake/stake.css:1]: Yeah, that's correct, ui should fit the figma. This is UI:
image

and this is Figma:
image


Please, fix it

**Review by whateverfw — CHANGES_REQUESTED** [review]
please fix unresolved issues

**Comment by djeck1432** [comment]
@canhamzacode Any updates? you will be unassigned soon 18 hours without update

**Comment by djeck1432** [comment]
@canhamzacode Texted you in telegram, waiting for changes or any updates

**Comment by djeck1432** [comment]
@canhamzacode Unassiged, has issue with laptop
